static/js/tools/timeline/util.ts

```diff
@@ -152,3 +153,35 @@ export function getDenom(mprop: string): string {
   }
   return "";
 }
+
+// Set metahash map (a map of stat var dcid to hash representing the source to
+// of data to use) in the URL
+export function setMetahash(metahash: Record<string, string>): void {
+  const urlParams = new URLSearchParams(window.location.hash.split("#")[1]);
+  let metahashMap = JSON.parse(urlParams.get("metahash"));
```

Comment:
can we start adding constants for the URL params? and maybe a shorter string here, `mh` or `meta`?

Reply:
done